Userland/Applications/Calendar/AddEventDialog.gml

```diff
@@ -102,6 +102,35 @@
         }
     }
 
+    @GUI::Widget {
+        fill_with_background_color: true
+        fixed_height: 25
+        layout: @GUI::HorizontalBoxLayout {
+            spacing: 4
+        }
+
+        @GUI::Label {
+            text: "Duration:"
+            text_alignment: "CenterLeft"
+            fixed_height: 14
+            font_weight: "Bold"
+        }
+
+        @GUI::SpinBox {
+            name: "duration_hour"
+            fixed_size: [50, 20]
+            min: 0
+            max: 999999
```

Comment:
better way to set maximum max?

Reply:
Not sure we have one. @AtkinsSJ what do you think? If you don't get a reasonable reply then this seems like it could make a reasonable issue to track in github.

Reply:
What exactly are you trying to do? If the goal is "limit the number to between 0 and INT_MAX", then you can just *not* specify `max` and it should already be that I think?

Reply:
Oh I just looked and no, the default max is 100. That's silly. I'd say, remove this, and then modify SpinBox to initialise the max value to the relevant INT_MAX.